roles/realm_join/tasks/install_pexpect.yml

```diff
@@ -0,0 +1,36 @@
+- name: Ensure pexpect to answer password
+  become: yes
+  package:
+    name: python3-pexpect
+    state: latest
+  # on Fedora this will be True due to the major version. Not nice, but works.
+  when:
+    - ansible_os_family == "RedHat"
+    - ansible_distribution_major_version is version('8', '>')
+
+- name: Ensure pexpect to answer password
+  become: yes
+  package:
+    name: python-pexpect
+    state: latest
+  when:
+    - ansible_os_family == "Debian"
+
+- name: Install pip to get supported pexpect version on RedHat < 8
+  become: yes
+  package:
+    name:
+      - epel-release
+      - python-pip
+    state: present
+  when:
+    - ansible_os_family == "RedHat"
+    - ansible_distribution_major_version is version('8', '<')
+
+- name: Ensure pexpect to answer password from pip
+  become: yes
+  pip:
```

Comment:
pexpect 2.3-11.el7 that is in EL7 is too old I guess?

Reply:
Yes, we need `>= 3.3` for the ansible module to work.